platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/CrlChecker.java

```diff
@@ -42,14 +48,16 @@
     private static String encryptionPropertiesLocation = new AbsolutePathResolver(
             "etc/ws-security/server/encryption.properties").getPath();
 
-    private CRL crl = null;
+    private AtomicReference<CRL> crlCache = new AtomicReference<>();
+
+    private CrlRefresh refresh = new CrlRefresh();
 
     /**
      * Constructor method. Reads encryption.properties and sets CRL location
      */
     public CrlChecker() {
-        Properties encryptionProperties = loadProperties(encryptionPropertiesLocation);
-        setCrlLocation(encryptionProperties.getProperty(CRL_PROPERTY_KEY));
+        Executors.newScheduledThreadPool(1)
+                .scheduleWithFixedDelay(refresh, 0, 1, TimeUnit.HOURS);
```

Comment:
✏️   `scheduleWithFixedDelay` is a static method on `Executors`, so:
`Executors.scheduleWithFixedDelay(refresh, 0, 1, TimeUnit.HOURS);`

--